concordium-node/src/configuration.rs

```diff
@@ -287,6 +287,13 @@ pub struct BakerConfig {
         env = "CONCORDIUM_NODE_BAKER_DECRYPT_CREDENTIALS"
     )]
     pub decrypt_baker_credentials: bool,
+    #[structopt(
+        long = "modules-cache-size",
+        help = "The maximum number of artifacts that can be stored in the module cache",
```

Comment:
I think this could be more useful. Maybe `smart contract modules` instead of `artifacts`. I am not sure that users will know what `artifact` means.